test/integration/util_test.go

```diff
@@ -0,0 +1,18 @@
+package test_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func startInstance() {
```

Comment:
I'd maybe call this `startCRC()` as the other functions are called `CheckCRCStatus`, etc. Not a strong suggestion, just mentioning it.

Reply:
BTW, there's a file called `utilities_test.go` where I put code like this: https://github.com/crc-org/crc/blob/e3054a158d9c6fad2e5641b7d5207a951287b576/test/integration/utilities_test.go. That said, recently we talked about moving this kind of code to a place which will be shared with e2e. I think I'm doing the same "start if not running" check here: https://github.com/crc-org/crc/blob/e3054a158d9c6fad2e5641b7d5207a951287b576/test/e2e/testsuite/testsuite.go#L583 Maybe if you put this in `extended` we can later refactor it to be used in multiple places. Again, up to you.